src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: 'Under construction',
+  reducers: {
+    checkInitialState: (state) => {
```

Comment:
- [ ] As requested by the previous reviewer in this [review link](https://github.com/jcmunav63/bookstore/pull/2#discussion_r1317552514), the name of the reducer action you are to create is the `checkStatus`, however what you currently have on `line 7` of the highlighted code is `checkInitialState`, please kindly correct this naming difference so that your project can be approved. Thank You, 👍 ✨